tools/build/scripts/utils/addPackageMetadata.js

```diff
@@ -7,9 +7,9 @@ const FS = require("fs-extra");
 const path = require("path");
 
 // We cannot guarantee the folder structure of a project
-// so find the project root using rush env variable if available.
-const rootPackageJson = process.env.RUSHSTACK_FILE_ERROR_BASE_FOLDER
-  ? path.join(process.env.RUSHSTACK_FILE_ERROR_BASE_FOLDER, "package.json")
+// so find the project root using require.main if available.
+const rootPackageJson = require.main
+  ? path.join(path.dirname(require.main.filename), "package.json")
```

Comment:
does this work? how did you test this out?

Reply:
So, I tested this locally, and it works as intended there, but it fails on Azure, and looking into this solution a bit more, that seems to be a consistent issue with this kind of code. Seems like a fatal issue, so I'm just going to close this PR and try to investigate some other solutions. May be best to just rename the environment variable after all.